schemas/com.vimeo/cue_point_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo Cue Point.",
```

Comment:
Not completely sure if this is true, but would be useful to have a description like that.

```suggestion
  "description": "Schema for a Vimeo cue point event fired when the playback position hits a registered cue point.",
```

Reply:
I've updated the description to something similar, hopefully it's descriptive enough.